tools/dataset_object_processor/dataset_object_processor.py

```diff
@@ -0,0 +1,879 @@
+import argparse
+import json
+import os
+import time
+from ntpath import basename
+from typing import Any, Dict, List, Optional, Tuple
+
+import git
+import magnum as mn
+import processor_utils as pcsu
+import psutil
+from colorama import Fore, init
```

Comment:
If you want to merge this PR into main, add `colorama` to the `requirements.txt` file

Reply:
cool will do

Reply:
It has specific workarounds for Windows, but is just a normal library for all other operating systems.